DataConnectors/SalesforceServiceCloud/SalesforceServiceCloud_API_FunctionApp.json

```diff
@@ -0,0 +1,130 @@
+{
+    "id": "SalesforceServiceCloud",
+    "title": "Salesforce Service Cloud (Preview)",
+    "publisher": "Salesforce",
+    "descriptionMarkdown": "The Salesforce Service Cloud data connector provides the capability to ingest information about your Salesforce operational events into Azure Sentinel through the REST API. The connector provides ability to review events in your org on an accelerated basis, get [event log files](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/event_log_file_hourly_overview.htm) in hourly increments for recent activity.",
+    "additionalRequirementBanner":"These queries and workbooks are dependent on a parser based on Kusto to work as expected. Follow the steps to use this Kusto functions alias **SalesforceServiceCloud** in queries and workbooks. [Follow steps to get this Kusto functions>](https://raw.githubusercontent.com/Azure/Azure-Sentinel/SalesforceServiceCloudConnector/Parsers/SalesforceServiceCloud/SalesforceServiceCloud) ",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "SalesforceServiceCloud_CL",
+            "baseQuery": "SalesforceServiceCloud_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Last Salesforce Service Cloud EventLogFile Events",
+            "query": "SalesforceServiceCloud\n | sort by TimeGenerated desc"
+        }
+     ],
+    "dataTypes": [
+        {
+            "name": "SalesforceServiceCloud_CL",
+            "lastDataReceivedQuery": "SalesforceServiceCloud_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "SalesforceServiceCloud_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
+        "isPreview": true
```

Comment:
Let's keep this in the json file so that it's retained and we are aware of it for future. This is not automatically added in GitHub - it needs to be manually committed.

Reply:
This is needed: "isPreview": true
We need to remove this from GitHub when it is GA'd.